locales/en/plugin__kubevirt-plugin.json

```diff
@@ -174,6 +174,7 @@
   "Automatically apply this key to any new VirtualMachine you create in this project.": "Automatically apply this key to any new VirtualMachine you create in this project.",
   "Automatically compute CPU limits on projects containing labels": "Automatically compute CPU limits on projects containing labels",
   "Automatically compute CPU limits on projects containing these labels": "Automatically compute CPU limits on projects containing these labels",
+  "Automatically pull updates from the RHEL repository.": "Automatically pull updates from the RHEL repository.",
```

Comment:
I am not sure if the "." should be ever there, at the end of this string. Because if you look at the other similar strings...

Reply:
There is no dot in the design. I'll remove it